no-vary-search.bs

```diff
@@ -45,9 +45,7 @@ spec: RFC8941; urlPrefix: https://www.rfc-editor.org/rfc/rfc8941.html
 
 <h2 id="status-and-venue">Status and venue note</h2>
 
-This document is being written as a web-style specification in the WICG for now, because that's the tooling and venue the author is familiar with. Its purpose is to nail down some details of the processing model in order to make writing and testing prototypes easier.
-
-In the longer term, we envision this header being specified in a HTTPWG RFC, alongside whatever portion of the processing model can be shared among its various consumers. (That is, between both web platform specifications such as [[FETCH]], and HTTP specifications such as future modifications to [[RFC9111]].) It's just incubating in WICG for now.
+Work on this document has moved to the IETF HTTP working group. The <a href="https://httpwg.org/http-extensions/draft-ietf-httpbis-no-vary-search.html">latest draft</a> and <a href="https://github.com/httpwg/http-extensions/labels/no-vary-search">issue tracker</a> can be found there.
 
 <h2 id="header-definition">HTTP header field definition</h2>
```

Comment:
I think we should gut this spec entirely, maybe replacing it with a (client-side) redirect.